mp/src/game/client/momentum/ui/PaintGunPanel.cpp

```diff
@@ -46,9 +47,9 @@ PaintGunPanel::PaintGunPanel() : BaseClass(g_pClientMode->GetViewport(), "PaintG
     m_pToggleSound->AddActionSignalTarget(this);
 
     m_pPickColorButton = new Button(this, "PickColorButton", "", this, "picker");
-    m_pSliderScale = new CvarSlider(this, "SliderScale");
+    m_pSliderScale = new CvarSlider(this, "SliderScale", "", 0.0f, 1.0f, "mom_paintgun_scale", false, true);
 
-    m_pTextSliderScale = new TextEntry(this, "TextSliderScale");
+    m_pTextSliderScale = new CvarTextEntry(this, "TextSliderScale", "mom_paintgun_scale");
```

Comment:
This should use the `%.2f` format as was used in the old `SetLabelText` method.

Reply:
See my comment [here](https://github.com/momentum-mod/game/pull/713#discussion_r417678682). 

I can add this here now then make that change in the PR where I change the `CvarSlider` & `CvarComboBox`, since that'd have to be another constructor change.